logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -71,34 +72,58 @@ function Get-DefaultDevelopmentSettingsByProject {
 
 function Get-CredentialSettingsByProject {
 
+    [CmdletBinding()]
```

Comment:
we don't want to modify this file. I was just using it as an example so you could see what the User object looked like in powershell.

Reply:
sure ..will modify and update now

Reply:
Reverted changes